packages/salesforcedx-vscode-core/src/commands/projectRetrieveStart.ts

```diff
@@ -55,13 +55,13 @@ export class ForceSourcePullExecutor extends SfdxCommandletExecutor<{}> {
     const builder = new SfdxCommandBuilder()
       .withDescription(nls.localize(this.params.description.default))
       .withArg(this.params.command)
-      .withJson()
+      .withJson(false)
```

Comment:
What is the difference between `.withJson(false)` and just completely taking out this line?

Reply:
The false in that method is to not include the --loglevel flag. If we remove the line we lose the --json flag as well